packages/flow-core/src/components/f-pictogram/f-pictogram.ts

```diff
@@ -136,6 +257,31 @@ export class FPictogram extends FRoot {
 			</div>
 		`;
 	}
+	protected async updated(
+		changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>
+	): Promise<void> {
+		super.updated(changedProperties);
+		colors = this.generateHslaColors(50, 60, 1.0, 10);
```

Comment:
This doesn't need to be part of class

So declare it outside of class and initialize colors variable with that function call.

Right now it is adding unnecessary call on every updated hook